.github/workflows/lighthouse.yml

```
@@ -89,13 +89,15 @@ jobs:
         with:
           path: ./build/*
           key: ${{ github.sha }}
+      - name: Install wp-env
+        shell: bash
+        run: |
+              npm i -g --no-package-lock --no-save @wordpress/env@8.6.1
```

Lets have a single place to specify `wp-env` in (`package.json`)
```suggestion
          npm ci
```

---

done